src/test/java/org/itsallcode/openfasttrace/cli/TestCliArguments.java

```diff
@@ -182,12 +182,23 @@ public void testSetI()
 
     // [utest->dsn~filtering-by-tags-during-import~1]
     @Test
-    public void testSetIngnoreTags()
+    public void testSetWantedTags()
     {
         final String value = "client,server";
         assertThat(BEFORE_SETTER, this.arguments.getWantedTags(), emptyIterable());
         this.arguments.setWantedTags(value);
         assertThat(AFTER_SETTER, this.arguments.getWantedTags(),
                 containsInAnyOrder("client", "server"));
     }
-}
+
+    // [utest->dsn~filtering-by-tags-or-no-tags-during-import~1]
+    @Test
+    public void testSetWantedTagsIncludingNone()
+    {
+        final String value = "_,client,server";
+        assertThat(BEFORE_SETTER, this.arguments.getWantedTags(), emptyIterable());
```

Comment:
You could move this to the setup method.

Reply:
Improved by moving to separate test.